src/gestures-hooks.tsx

```diff
@@ -13,29 +13,14 @@ interface RecordingData {
   data: XYZData;
 }
 
-interface StoredGestureData {
+export interface Gesture {
   name: string;
   ID: number;
-  recordings: RecordingData[];
-}
-export interface GestureData extends StoredGestureData {
-  // Confidence is added after a successful training and predicting of data
-  confidence?: {
-    currentConfidence?: number; // from 0-1
-    requiredConfidence: number; // from 0-1
-  };
-}
-
-// Used for getPredicted in MlActions hook
-export interface ConfidentGestureData extends StoredGestureData {
-  confidence: {
-    currentConfidence: number; // from 0-1
-    requiredConfidence: number; // from 0-1
-  };
+  requiredConfidence?: number;
```

Comment:
There's not really consensus that this belongs here. I think we should pull more people into the discussion but go with this for now. The current prototype just loses your threshold setting all the while.